diff --git a/node_modules/three/examples/jsm/postprocessing/OutlinePass.js b/node_modules/three/examples/jsm/postprocessing/OutlinePass.js
index 4d0fc55..3a7c480 100644
--- a/node_modules/three/examples/jsm/postprocessing/OutlinePass.js
+++ b/node_modules/three/examples/jsm/postprocessing/OutlinePass.js
@@ -220,7 +220,7 @@ class OutlinePass extends Pass {
 
 		function VisibilityChangeCallBack( object ) {
 
-			if ( object.isMesh || object.isSprite ) {
+			if ( object.isMesh || object.isSprite || object.isLine || object.isTransformControls ) {
 
 				// only meshes and sprites are supported by OutlinePass
 
